docs/searching-data.md

```diff
@@ -0,0 +1,71 @@
+# Searching Data
+To derive the insights you want from Gro's data, you will first want to find the data you're interested in. Below you will find some of the most useful tips on how to discover the data of greatest value to you.
+
+## Code snippets
+Users may find that the Gro API is at its most powerful when used in conjunction with the Gro web application. The web app provides the most convenient format for selecting the data series that is of most interest to you. In our Add Data Series window, you can select entities of interest, and then other entities for which no data is available based on your selection will be filtered out from the remaining options. For example, after selecting the item "Corn" only metrics and regions that have data for 'Corn' will remain selectable.
+![add-data-series-example.png](./_images/add-data-series-example.PNG)
+
+Once you have created a chart with data that you want, you can take advantage of our Code Snippets feature to pull that data into your API client code.
+
+Simply click on the Chart Dropdown Button, then select Export and API Client Code Snippets. 
+![code-snippet-dropdown.png](./_images/code-snippet-dropdown.PNG)
+
+Every unique data series from your chart will now be available in a client.get_data_points function for easy copying and pasting into your code or command line.
+![code-snippet-copy-code.png](./_images/code-snippet-copy-code.PNG)
+
+For charts that have multiple data series, you have the option to Select all unique data series, or to select the individual series that are of greatest interest to you. 
+![code-snippet-select-all.png](./_images/code-snippet-select-all.PNG)
+
+## Search
+As described in the [Data Series Definition](./data-series-definition.md) page, a data series in Gro is a unique combination of the entities: item, metric, region, partner_region (optional), frequency, and source. To find the specific entity you would like to retrieve data for, you can use a variety of search methods. For example, [`client.search()`](https://gro-intelligence.github.io/api-client/api.html#api.client.lib.search) will return a list of IDs that match your search term. If you want to understand the differences between various search results, you may find the [`client.search_and_lookup()`](https://gro-intelligence.github.io/api-client/api.html#api.client.lib.search_and_lookup) method more helpful.

```

Comment:
Since the API reference page was updated to be all at the GroClient level, these links including `api.client.lib` don't work any more

Doing a command-f on this PR I see 6 instances of this:

```
api.html#api.client.lib.
```

should become

```
api#api.client.gro_client.GroClient.
```

Reply:
Also shouldn't include https://gro-intelligence.github.io/api-client/ in the links. Should be relative links. So:

`(./api#api.client.gro_client.GroClient.search)`

Reply:
done, and good catch